scripts/bootstrap-aio.sh

```diff
@@ -63,16 +63,6 @@ if ! container_artifacts_available; then
   ./scripts/artifacts-building/remove-container-aio-config.sh
 fi
 
-# Run AIO bootstrap playbook
```

Comment:
These need to stay - it actually an upstream bug which hasn't been dealt with yet IIRC.

Reply:
these are the default settings in the RC file. Do they still need to be here if we're not modifying that pathing?

Reply:
Yes, sorry - I got a little confused. This should be able to go.